libs/payments/ui/src/lib/client/components/AlertBar/index.tsx

```diff
@@ -19,58 +20,52 @@ export enum AlertBarVariant {
 export type AlertBarProps = {
   checked?: boolean;
   children: React.ReactNode;
-  ariaId?: string;
   variant?: AlertBarVariant;
-  onClick?: () => any;
   containerRef?: HTMLElement | null;
 };
 
 export const AlertBar = ({
   checked,
   children,
   variant = AlertBarVariant.ERROR,
-  ariaId,
-  onClick,
   containerRef = null,
 }: AlertBarProps) => {
-  const [container, setContainer] = useState<HTMLElement | null>(null);
+  const [openDialog, setOpenDialog] = useState(true);
 
-  useEffect(() => {
-    if (!containerRef) {
-      setContainer(document.getElementById('header'));
-    }
-  }, []);
+  const { l10n } = useLocalization();
 
   let alertTypeStyle;
   switch (variant) {
     case AlertBarVariant.ERROR:
-      alertTypeStyle = 'bg-red-600 text-white';
+      alertTypeStyle = 'bg-red-100 error';
```

Comment:
note: Matching the styles used by FxA. This was a suggestion made by UX/Product in [PAY-3239](https://mozilla-hub.atlassian.net/browse/PAY-3239?filter=15915)